compression_verifier.go

```diff
@@ -21,12 +21,10 @@ const (
 )
 
 type (
-	columnCompressionConfig map[string]string
```

Comment:
Removed this type because it is only used once and it's also now a part of the public API.